en/telemetry/esp32_wifi_module.md

```diff
@@ -1,169 +1,159 @@
+- Set the autopilot port to MAVLINK 1 or 2 protocol.
 
 ::: info
 
-- Follow the ESP32 board manufacturers recommendations on power supply.
+- Follow the ESP32 board manufacturer recommendations on power supply.
   Some boards might have issues if they are simultaneously connected to a 5V power source and have a USB cable connected to the USB/Serial bridge (USB socket of the ESP32 dev board).
 - Some ESP32 DevKits manufacturers use the wrong labels for the pins on their products.
-  Make sure that the PINs on your board are labeled correctly if you encounter issues.
+  Make sure that the PINs on your board are labelled correctly if you encounter issues.
 
 :::
 
-## Configuring QGroundControl
+## Configuring PX4
 
-QGroundControl should auto-detect the connection and no further actions should be necessary (_DroneBridge for ESP32_ automatically forwards data from all connected WiFi devices via UDP to port 14550).
+Configure the Flight controller port that is wired to the ESP32 to have MAVLink set as the protocol and to have the same baud rate as the ESP32.
+The Pixhawk `TELEM1` and `TELEM2` ports are configured for [MAVLink by default](../peripherals/mavlink_peripherals.md#default-mavlink-ports) but you will still need to set the rates.
 
-The following connection options are available:
+For `TELEM1` you might set the values like this:
 
-- UDP unicast on port `14550` to all connected devices.
-- TCP on port `5760`
+- `SER_TEL1_BAUD`: 115200 8N1
+- `MAV_0_RATE`: 24000 (can be increased optionally)
+
+For more information, and using other ports, see [MAVLink Peripherals](../peripherals/mavlink_peripherals.md) and [Serial Port Configuration](../peripherals/serial_configuration.md)
 
 ## Configuring DroneBridge for ESP32
 
 The _DroneBridge for ESP32_ default configuration should work for connecting to PX4 "out of the box".
+For the Official HW1.x Boards the configuration is: `UART TX Pin: 5`, `UART RX Pin: 4`, `UART RTS Pin: 6`, `UART CTS Pin: 7`
+
 The only configuration that may be required is ensuring that the baud rates of the ESP32 and flight controller match.
 
+![DroneBridge for ESP32 Webinterface](../../assets/peripherals/telemetry/esp32/dbesp32_webinterface.png)
+
 You will want to change these settings if you want to use different pins on the ESP32, a different WiFi configuration, or to tweak the packet size.
-Lower packet size means more overhead and load on the system but also less latency and faster recovery from a lost packet.
+A lower packet size means more overhead and load on the system, less latency, and faster recovery from a lost packet.
 
-### Default Configuration
 
-- SSID: `DroneBridge for ESP32`
-- Password: `dronebridge`
-- Transparent/MAVLink
-- UART baud rate `115200`
-- UART TX pin `17`
-- UART RX pin `16`
-- Gateway IP: `192.168.2.1`
 
-### Custom Settings & Webinterface
+### Supported Modes
 
```

Comment:
These are ESP32 modes? I presume this is the "ESP32 mode? on the screen shown above? Does that mean we don't need the MAVLink index in that table for anything? Should it be removed?

Reply:
Yes these are available via the web interface. Most of the parameters can also be changed via the GCS and the MAVLink parameter protocol instead of the web interface. In that case the MavLink Index is the value to set SYS_ESP32_MODE to.
I added some more description.